lib/jsonapi/rails/renderer.rb

```diff
@@ -28,6 +28,8 @@ def self.render(errors, options)
     # @api private
     def rails_renderer(renderer)
       proc do |json, options|
+        reverse_mapping = request.env[ActionController::REVERSE_MAPPING_KEY]
```

Comment:
Would you mind adding a comment stating that the proc is instance_eval'd in the controller context? It will simplify debugging in case this assumption is broken by rails at some point.